README.md

```diff
@@ -1811,6 +1811,7 @@ An [integrated development environment (IDE)](https://en.wikipedia.org/wiki/Inte
 - [ML Workspace](https://github.com/ml-tooling/ml-workspace) - All-in-one web-based IDE for machine learning and data science. `Apache-2.0` `Docker`
 - [Motor Admin](https://www.getmotoradmin.com/) - No-code admin panel and business intelligence software - search, create, update, and delete data entries, create custom actions, and build reports. ([Demo](https://motor-admin.herokuapp.com/demo/), [Source Code](https://github.com/motor-admin/motor-admin)) `AGPL-3.0` `Ruby`
 - [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) - Web browser based IDE for R. ([Source Code](https://github.com/rstudio/rstudio)) `AGPL-3.0` `Java/C++`
+- [Resoto] (https://resoto.com) - Resoto is an open source cloud asset inventory. Write code that reacts to changes in your cloud infrastructure. Resoto helps you search infrastructure, generate reports, and automate tasks. ([Source Code](https://github.com/someengineering/resoto)) `Apache-2.0` `Python/Docker`
```

Comment:
- Please remove the extra space between [Resoto] and (https://resoto.com)

> https://resoto.com/docs#what-is-resoto

-  Please change the description to  `Create indexes resources, captures dependencies, and maps out your infrastructure in an intuitive [graph](https://resoto.com/docs/concepts/graph). The graph contains metrics for each [resource](https://resoto.com/docs/reference/data-models).`